.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+-Xmx2048m
```

Comment:
@batmat Something is wrong in the Jenkins build - it should not need to use anywhere near that much memory :(
I've had far more complex builds use just 120MB.

800MB sounds like it is reasonable interim solution until this is tracked down.